content/agent/kubernetes/integrations.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Custom Integrations
+kind: documentation
+---
+
+## ConfigMap
+It is possible to leverage the ConfigMaps to configure or enable integrations.
+To do so, you only need to create a ConfigMap with the integration(s)'s configuration.
+Then, reference this ConfigMap among the volumes of your Agent's manifest.
+
+For example, in the following case we customize the name, url and tags fields of the http check.
+To enable other integrations, just specify the correct yaml name and make sure it is properly formated.
```

Comment:
Is it worthwhile to include or link to the http check YAML file so that users can quickly see how it compares with the ConfigMap YAML?